clients/devtools-report-assets.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * @license Copyright 2019 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+/* global Runtime */
+
+// @ts-ignore: Runtime exists in devtools.
+const cachedResources = Runtime.cachedResources;
```

Comment:
could we get a fileOverview comment that explains this is basically our shim in devtools land for our own `html-report-assets` file?

might be even worth adding an extra note in `html-report-assets`  that changes there will have to be mirrored on this end?